app/index.html

```diff
@@ -129,6 +129,66 @@ <h2 class="card-pf-title">
             </div>
 
           </div>
+          <!-- stack analyses-->
+          <div class="row row-cards-pf">
+            <div class="card-pf card-pf-accented">
+              <div class="card-pf-heading">
+                <h2 class="card-pf-title">
+                  Stack Analyses
+                </h2>
+              </div>
+              <div class="card-pf-body">
+                <div id="stackAnalyses">
+                  <div class="row" id="pomTextConetntArea">
+                    <div class="col-lg-12 col-sm-12">
+                      <textarea class="form-control" id="pomTxtArea" placeholder="Enter POM.xml content here"></textarea>
+                    </div>
+                    <div class="col-lg-12 col-sm-12">
+                      <br />
+                      <button type="button" id="stacAnalyseskbtn" class="btn btn-primary">Submit</button>
+                    </div>
+                  </div>
+                  <br />
+                  <div id="pomStatusSuccess" class="alert alert-success alert-dismissable">
+                    <span class="pficon pficon-ok"></span>
+                    <strong>Great!</strong> Stack ID has been created. <a href="#" id="stackAnalysesAnchor" class="alert-link">View report</a>.
```

Comment:
changed it to message passed from API , as there is lag between stackID generation and report creation. couple of secs to minutes .